installer/targets/ros-python-sip/install.bash

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+local SIP_version="4.15.5"
+
+if [ $(lsb_release -sc) == "trusty" ]
+then
+    tue-install-system python-sip-dev
+else
+
+    if command -v sip >/dev/null 2>&1
+    then
+        # We have SIP, let check is version
+        SIP_installed=$(sip -V)
+        if [[ "$SIP_installed" == "$SIP_version" ]]
+        then
+            # SIP is correct version, so don't need to install anything
+            return
+        fi
+    fi
+    # We don't have SIP or not the correct version
+    SIP_file=~/.tue/installer/targets/ros-python-sip/sip-$SIP_version.tar.gz
+    if [ ! -f "$SIP_file" ]
+    then
+        url="https://downloads.sourceforge.net/project/pyqt/sip/sip-$SIP_version/sip-$SIP_version.tar.gz"
+        tue-install-error "Download $url and place it in ~/.tue/installer/targets/ros-python-sip/"
+        return -1
+    fi
+    
+    tue-install-info "Installing SIP version: $SIP_version"
+    cp $SIP_file /tmp/sip.tar.gz
+
+    tar xvzf /tmp/sip.tar.gz -C /tmp
+    cd /tmp/sip-$SIP_version
+    python ./configure.py
+    make
+    sudo make install
```

Comment:
Please don't use `sudo make install` because it makes it impossible to remove the package again. Use the following:
```
sudo checkinstall -y --nodoc make install
```

Reply:
When using ```sudo checkinstall -y --nodoc make install``` the following log shows:
```
(Reading database ... 469674 files and directories currently installed.)
Preparing to unpack .../sip_4.15.5-1_amd64.deb ...
Unpacking sip (4.15.5-1) ...
dpkg: error processing archive /tmp/sip-4.15.5/sip_4.15.5-1_amd64.deb (--install):
 trying to overwrite '/usr/bin/sip', which is also in package sip-dev 4.17+dfsg-1build1
dpkg-deb: error: subprocess paste was killed by signal (Broken pipe)
Errors were encountered while processing:
 /tmp/sip-4.15.5/sip_4.15.5-1_amd64.deb
```
The log is shown in a editor, so you have to close that one to continue the installer.

But afterwards ```sip -V``` does provide the correct version.

Reply:
@Rayman I don't see a correct use of `checkinstall`. Do know any solution?

Reply:
```
 trying to overwrite '/usr/bin/sip', which is also in package sip-dev 4.17+dfsg-1build1
```
This is good, dpkg keeps track of files so when you uninstall packages, all the files are removed. You are not able to uninstall sip when you did `sudo make install`

Reply:
But it is already overwritten by `checkinstall`:
```
make[1]: Entering directory '/tmp/sip-4.15.5/sipgen'
cp -f sip /usr/bin/sip
```
Therefore the use of `checkinstall` is not really usefull